Remotion/Data/DomainObjects/Utilities/ObjectIDConverter.cs

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using Remotion.Utilities;
+
+namespace Remotion.Data.DomainObjects.Utilities;
+
+public class ObjectIDConverter : TypeConverter
+{
+  private readonly Type _stringType = typeof(string);
+
+  public override bool CanConvertFrom (ITypeDescriptorContext? context, Type sourceType)
+  {
+    ArgumentUtility.CheckNotNull(nameof(sourceType), sourceType);
+
+    return sourceType == _stringType;
```

Comment:
I don't know if we need it but in `WebStringConverter` we also did base calls as `TypeConverter` has special handling for `InstanceDescriptor`. It's probably not necessary but I would add the base calls for consistency. Also affects other methods as well.

Reply:
I concur with consistency.

Reply:
Hmm... for practical purposes, we don't need to worry about `InstanceDescriptor`. It's used with the WebForms designer etc. 

If you want to handle it, you can do it identically to PlainTextStringConverter implementation, the same semantics apply for both cases (a string representation gets convertered to an object).